src/components/tooltip/Tooltip.tsx

```diff
@@ -266,7 +269,7 @@ class Tooltip extends React.Component<TooltipProps, State> {
             <TetherComponent ref={this.tetherRef} {...tetherProps}>
                 {React.cloneElement(React.Children.only(children) as React.ReactElement, componentProps)}
                 {showTooltip && (
-                    <div className={classes} id={this.tooltipID} role="tooltip">
+                    <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
```

Comment:
Technically I don't think `aria-live` is supposed to be used on elements that are dynamically added/removed from the DOM. 

From MDN (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions): 
"Dynamically adding an element with an aria-live attribute or specialized role to the document also won't result in any announcement by assistive technologies (as, at that point, the browser/assistive technologies are not aware of the live region yet, so cannot monitor it for changes).

Always make sure that the live region is present in the document first, and only then dynamically add/change any content."

But if this makes it read it out, then 🤷‍♂ ...

Reply:
I tested without `aria-live="polite"` on the tooltip and it fails ChromeVox from reading the tooltip. NVDA and JAWS work without `aria-live="polite"`.